src/Controller/AdminUser.php

```diff
@@ -13,12 +15,26 @@ class AdminUser extends \Symfony\Bundle\FrameworkBundle\Controller\AbstractContr
 {
 
     /**
+     * @param Request $request
+     * @param User $user
      * @return Response
      * @IsGranted("ROLE_USER", statusCode=404)
      * @Route("/mon-compte/{firstname}/{lastname}", name="user")
      */
-    public function myAccount(): Response
+    public function myAccount(Request $request, User $user): Response
```

Comment:
pas la pein de passer un param $user
tu récup direct $this->getUser() dans la méthode et tu as l'utilsateur actuellement connecté

Reply:
voire encore mieux, tu ne recup pas du tout user et tu utilises app.user dans Twig (ce que tu fais d'ailleurs donc ton $user te sers vraiment à rien ;-))